osbrain/agent.py

```diff
@@ -898,6 +898,51 @@ def _connect_and_register(self, client_address, alias=None, handler=None,
         self.register(socket, register_as, alias, handler)
         return client_address
 
+    def unsubscribe_from_topic(self,
+                               socket: Union[AgentAddress, str, zmq.Socket],
```

Comment:
Maybe `Union[AgentAddress, AgentChannel, str]`? `AgentChannel should be there, and I would remove `zmq.Socket`, as the user should never be working with raw sockets in osBrain.